frontend/src/old-pages/Clusters/Properties.tsx

```diff
@@ -100,36 +101,46 @@ export default function ClusterProperties() {
                     triggerType="custom"
                     content={
                       <StatusIndicator type="success">
-                        ARN copied
+                        {t('cluster.properties.sshcommand.success')}
                       </StatusIndicator>
                     }
                   >
                     <Button
                       variant="inline-icon"
                       iconName="copy"
-                      ariaLabel="Copy ARN"
+                      ariaLabel={t('cluster.properties.sshcommand.help')}
                       onClick={() => {
                         navigator.clipboard.writeText(
-                          cluster.cloudformationStackArn,
+                          `ssh ${clusterDefaultUser(cluster)}@${headNode.publicIpAddress}`
                         )
                       }}
                     >
                       copy
                     </Button>
                   </Popover>
                 </Box>
-                <Link
-                  external
-                  href={`${consoleDomain(
-                    cluster.region,
-                  )}/cloudformation/home?region=${
-                    cluster.region
-                  }#/stacks/events?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false&stackId=${
-                    cluster.cloudformationStackArn
-                  }`}
-                >
-                  {cluster.cloudformationStackArn}
-                </Link>
+                ssh {clusterDefaultUser(cluster)}@{headNode.publicIpAddress}
+                <HelpTooltip>
+                  <Trans i18nKey="cluster.properties.sshcommand.tooltiptext">
+                    <a
+                      rel="noreferrer"
+                      target="_blank"
+                      href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html#AccessingInstancesLinuxSSHClient"
+                    ></a>
+                  </Trans>
+                </HelpTooltip>
+                {/* <HelpTooltip>
```

Comment:
todo: just remove the comment and it's ready to be merged!

Reply:
done!